programs/server/config.xml

```diff
@@ -1336,17 +1336,13 @@
         name - name for the rule (optional)
         regexp - RE2 compatible regular expression (mandatory)
         replace - substitution string for sensitive data (optional, by default - six asterisks)
-    -->
     <query_masking_rules>
         <rule>
             <name>hide encrypt/decrypt arguments</name>
             <regexp>((?:aes_)?(?:encrypt|decrypt)(?:_mysql)?)\s*\(\s*(?:'(?:\\'|.)+'|.*?)\s*\)</regexp>
-            <!-- or more secure, but also more invasive:
-                (aes_\w+)\s*\(.*\)
-            -->
             <replace>\1(???)</replace>
         </rule>
-    </query_masking_rules>
+    </query_masking_rules> -->
```

Comment:
Why do we want to comment `query_masking_rules` ?

Reply:
Those masking rules were written to replace arguments of encrypt functions in logs with `???`.
But now with my PR those arguments are always hidden in logs and so it's not necessary to specify that in the configuration anymore.